internal/api/keppel/accounts.go

```diff
@@ -542,6 +544,66 @@ func (a *API) handlePutAccount(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		// Copy PlatformFilter when creating an account with the Replication Policy on_first_use
+		if req.Account.ReplicationPolicy != nil {
```

Comment:
This branch should only be taken when an account is being created, i.e. `if account == nil` (in other words, it should go in the `if account == nil` that ended just above these lines).